lib/config/target.js

```diff
@@ -194,7 +194,7 @@ You can also more options via the 'target' option: 'browserslist' / 'browserslis
 				destructuring: v(6),
 				bigIntLiteral: v(10, 4),
 				dynamicImport: v(12, 17),
-				dynamicImportInWorker: major ? false : undefined,
+				dynamicImportInWorker: v(12, 17),
```

Comment:
Can you provide a link to where you found these versions? Just make sure we have the right versions

I looked at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import

> Node.js 13.2 (Release date: 2019-11-21)
> Full support